community/command-line/src/main/java/org/neo4j/commandline/admin/AdminCommand.java

```diff
@@ -81,6 +81,11 @@ public String name()
         public abstract String summary();
 
         /**
+         * @return AdminCommandSection the command using the provider is grouped under
+         */
+        public abstract AdminCommandSection commandSection();
```

Comment:
I thought it might be useful to make people think about sections when they add new commands. We have a `General` section for commands that can't be grouped to a specific section.